### PR TITLE
Run from device from bundle

### PR DESCRIPTION
**Pull request by ivanbogdanov**

### CODE REVIEW

File: app/ios/SmartSyncExplorerReactNative.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -269,7 +271,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.4;
+				IPHONEOS_DEPLOYMENT_TARGET = 8.2;
```
Comment (kchitalia) [inline app/ios/SmartSyncExplorerReactNative.xcodeproj/project.pbxproj:274]: Any particular reason to bump this down to 8.2?

File: app/ios/SmartSyncExplorerReactNative.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -307,7 +309,7 @@
...
```
Comment (kchitalia) [inline app/ios/SmartSyncExplorerReactNative.xcodeproj/project.pbxproj:312]: Same here.

File: app/ios/SmartSyncExplorerReactNative.xcodeproj/project.pbxproj (hunk 1 above)
Comment (ivanbogdanov) [inline app/ios/SmartSyncExplorerReactNative.xcodeproj/project.pbxproj:274]: No reason actually: it just works with 8.2

File: app/ios/SmartSyncExplorerReactNative.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -166,26 +169,25 @@
...
 			inputPaths = (
 			);
-			name = "Embed Pods Frameworks";
 			outputPaths = (
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-SmartSyncExplorerReactNative/Pods-SmartSyncExplorerReactNative-frameworks.sh\"\n";
-			showEnvVarsInLog = 0;
+			shellScript = "curl http://localhost:8081/index.ios.bundle -o main.jsbundle";
```
Comment (bhariharan) [inline app/ios/SmartSyncExplorerReactNative.xcodeproj/project.pbxproj:190]: What is this?

Comment (bhariharan) [inline app/ios/SmartSyncExplorerReactNative.xcodeproj/project.pbxproj:190]: Never mind. Got it.

**Comment by huminzhi** [comment]
@bhariharan @ivanbogdanov can we solve or close this pr? Thanks!

**Pull request closed without merging**